docs/userguides/testing.md

```diff
@@ -21,18 +21,19 @@ Tests are generally divisible into three parts:
 3. Assertion
 
 In the example above, we created a fixture that deploys our smart-contract. This is an example of a 'setup' phase.
-Next, we need to call a method on our contract. Let's assume there is a method called `is_owner()` that returns `True`
-when it is the owner of the contract making the transaction.
+Next, we need to call a method on our contract. Let's assume there is an `authorized_method()` that requires the
```

Comment:
@fubuloubu told me a while back to put each sentence on a newline in our markdown docs, and that is how I have been doing it since. Perhaps this is something we should officially decide on and put in contributing guide.